src/Mechanical/Rotational/components.jl

```diff
@@ -107,4 +126,38 @@ function IdealGear(;name, ratio, use_support=false)
         0 ~ ratio*flange_a.tau + flange_b.tau
     ]
     extend(ODESystem(eqs, t, sts, [ratio]; name=name), partial_element)
+end
+
+"""
+    RotationalFriction(;name, f=0.001, tau_c=20, w_brk=0.1, tau_brk=25) 
+
+Models rotational friction with Stribeck friction, Coulomb friction and viscous friction between the two flanges.
+The friction torque is a function of the relative angular velocity between flange_a and flange_b.
+
+Friction model: "Armstrong, B. and C.C. de Wit, Friction Modeling and Compensation, The Control Handbook, CRC Press, 1995."
+
+# States:
+- see [`PartialCompliantWithRelativeStates`](@ref)
+
+# Connectors:
+- See [`PartialCompliantWithRelativeStates`](@ref)
+
+# Parameters:
+- `f`: [`N⋅m/(rad/s)`] Viscous friction coefficient 
+- `tau_c`: [`N⋅m`] Coulomb friction torque
+- `w_brk`: [`rad/s`] Breakaway friction velocity 
+- `tau_brk`: [`N⋅m`] Breakaway friction torque
+"""
+function RotationalFriction(;name, f=0.001, tau_c=20, w_brk=0.1, tau_brk=25) 
```

Comment:
I don't think it's reasonable to have default values here, they all feel very arbitrary and will with a close to 100% probability need to be selected manually.
```suggestion
function RotationalFriction(;name, f, tau_c, w_brk, tau_brk) 
```
In general, arbitrary default values may hide modeling bugs, I prefer getting an
```
ERROR: UndefKeywordError: keyword argument tau_brk not assigned
```
rather than having the wrong value chosen for me.